ISISScientificComputing/autoreduce
# Add icat as pypi library

### Summary of work
* Add ICAT to setup.py 
* Remove ICAT installation instructions and scripts 

### How to test your work
* create a new virtualenv
* pip install -e autoreduction
* ensure the icat_client tests run successfully

### Additional comments
* This is technically validated by the travis build

Fixes #445
